package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "melosys-kodeverk",
-  "version": "5.15.15",
+  "version": "5.15.16",
```

Comment:
Trenger ikke oppdatere versjon her

Reply:
Jeg overskrev den committen for en uke siden, men PR-en viser ikke den nye committen hvor jeg har fjernet oppdatering av versjon.. 🤔